<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransplantDialog</class>
 <widget class="QDialog" name="TransplantDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transplant Changesets</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&amp;Revisions:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="buddy">
      <cstring>revisionsEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPlainTextEdit" name="revisionsEdit">
     <property name="toolTip">
      <string>Enter changesets by number, id, range or revset expression one per line</string>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Repositor&amp;y:</string>
     </property>
     <property name="buddy">
      <cstring>repoEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="repoEdit">
     <property name="toolTip">
      <string>Enter a repository URL, if changesets shall be transplanted from a repository</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Branch:</string>
     </property>
     <property name="buddy">
      <cstring>branchesCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="branchesCombo">
     <property name="toolTip">
      <string>Select or enter the branch to transplant from</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="allCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Select to transplant all changesets of the branch</string>
     </property>
     <property name="text">
      <string>Transplant &amp;All Changesets</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&amp;Skip:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="buddy">
      <cstring>pruneEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPlainTextEdit" name="pruneEdit">
     <property name="toolTip">
      <string>Enter changesets by number, id, range or revset expression one per line</string>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&amp;Merge:</string>
     </property>
     <property name="buddy">
      <cstring>mergeEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="mergeEdit">
     <property name="toolTip">
      <string>Enter revisions to be merged separated by a space character</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="logCheckBox">
     <property name="toolTip">
      <string>Select to append transplant info to the log message</string>
     </property>
     <property name="text">
      <string>Append Transplant &amp;Info</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>revisionsEdit</tabstop>
  <tabstop>repoEdit</tabstop>
  <tabstop>branchesCombo</tabstop>
  <tabstop>allCheckBox</tabstop>
  <tabstop>pruneEdit</tabstop>
  <tabstop>mergeEdit</tabstop>
  <tabstop>logCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TransplantDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TransplantDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
